docs/source/how_to_guides/configure_cloud_storage_cred.md

```diff
@@ -56,7 +56,9 @@ For [MosaicML platform](https://www.mosaicml.com/cloud) users, follow the steps
 
 ### Others
 
-Streaming dataset supports [GCP user credentials](https://cloud.google.com/storage/docs/authentication#user_accounts).  Users must set their GCP `user access key` and GCP `user access secret` in the run environment.
+Streaming dataset supports [GCP user credentials](https://cloud.google.com/storage/docs/authentication#user_accounts) or [HMAC keys for User account](https://cloud.google.com/storage/docs/authentication/hmackeys).  Users must set their GCP `user access key` and GCP `user access secret` in the run environment.
+
+From Google Cloud Console Navigate to `Google Storage` > `Settings (Left vertical pane)` > `Interoperability` > `Service account HMAC` > `User account HMAC` > `Access keys for your user account` > `Create a key`.
```

Comment:
From the Google Cloud console, navigate

Reply:
Addressed in https://github.com/mosaicml/streaming/pull/181/commits/4f5a1d61ae90a392a183a3380cec6c3814dc9adf.